api/app/signals/apps/signals/migrations/0095_SIG-2224_SIG-2234_new_categories.py

```diff
@@ -0,0 +1,86 @@
+"""
+Added new categories according to tickets SIG-2224 & SGI-2234
+"""
+
+from django.db import migrations
+
+NEW_CATEGORIES = {
+    # SIG-2224
+    'civiele-constructies': {
+        'sluis': {  # SIG-2224
+            'name': 'Sluis',
+            'handling': 'URGENTE_MELDINGEN',
+            'slo': '5W',
+            'departments': [
+                'STW',
+                'VOR',
+            ]
+        },
+        'watergangen': {  # SIG-2224
+            'name': 'Watergangen',
+            'handling': 'URGENTE_MELDINGEN',
+            'slo': '5W',
+            'departments': [
+                'STW',
+                'VOR',
+            ]
+        },
+        'afwatering-brug': {  # SIG-2224
+            'name': 'Afwatering brug',
+            'handling': 'URGENTE_MELDINGEN',
+            'slo': '5W',
+            'departments': [
+                'STW',
+                'VOR',
+            ]
+        },
+    },
+    # SIG-2234
+    'wegen-verkeer-straatmeubilair': {
+        'omleiding': {  # SIG-2234
+            'name': 'Omleiding',
+            'handling': 'A3WEC',
```

Comment:
```python
    Category.HANDLING_A3WEC: """
Wij handelen uw melding binnen drie weken af. U ontvangt dan geen apart bericht meer.
En anders hoort u - via e-mail - wanneer wij uw melding kunnen oppakken.""",
```

Bovenstaand bericht (uit `messages.py`) wijkt af van het bericht in ticket  2234:
"""
Wij handelen uw melding binnen drie weken af. U ontvangt dan geen apart bericht meer.
"""